### PR TITLE
Setup controllers

### PR DESCRIPTION
**Pull request by limavfabio**
This PR setups controllers and wireframes for views for the home index, users show page, posts index and show page.

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def index
+    @users = User.all
+  end
+end
```
Comment (Qoosim) [inline app/controllers/home_controller.rb]: - [ ] You're only required to create users and posts controller with index and show actions in both of them. Kindly take this index action to the users controller and remove this homeController

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:user_id])
+  end
+end
```
Comment (Qoosim) [inline app/controllers/users_controller.rb]: - [ ] It is required to have index and show actions in your usersController according to the project requirement.

### All the users (index)
all-users

### show action
single-user

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
+  get 'home/index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  root 'home#index'
+  get 'user/:user_id', to: 'users#show', as: :user
+  get 'user/:user_id/posts', to: 'posts#index', as: :user_posts
+  get 'user/:user_id/posts/:id', to: 'posts#show', as: :user_post
```
Comment (Qoosim) [inline config/routes.rb]: - [x] It is always adviceable to make the routes clear and clean. Kindly use below code to route your posts inside the users.

```
     resources :users, only: [:index, :show] do
         resources :posts, only: [:index, :show]
     end

```

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @limavfabio, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/home_controller.rb (hunk 1 above)
Comment (limavfabio) [reply app/controllers/home_controller.rb]: Without a HomeController, how will I have a root page?

Comment (limavfabio) [reply app/controllers/home_controller.rb]: On the wireframe, there isn't a /users route, the users are listed on the root route

Comment (limavfabio) [reply app/controllers/home_controller.rb:5]: On the wireframe, there isn't a /users route, the users are listed on the root route

File: config/routes.rb [hunk 4]
```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  root 'users#index'
+
+  resources :users, only: [:show] do
```
Comment (Ismailco) [inline config/routes.rb]: - [ ] Nice work using resources for your routes, however, the `:users` route should include `index` and ` show`, kindly consider adding `index` to this route

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @limavfabio, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used gitflow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @limavfabio , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested change. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**